packages/@okta/i18n/src/properties/login.properties

```diff
@@ -878,6 +878,8 @@ oie.duo.authenticator.description = Verify your identity using Duo Security.
 oie.duo.enroll.title = Set up Duo Security
 oie.duo.verify.title = Verify with Duo Security
 oie.duo.iFrameError = Error loading Duo. Try again or contact your admin for assistance.
+oie.authenticator.duo.method.duo.verification_timeout = We were unable to verify with Duo. Try again.
```

Comment:
Curious if we know why the backend sending different i18n keys for the same error message ?

Reply:
I think that was running by @ManuMalhotra-okta and @vincentvoong-okta. To me, I still think we can provide different message on backend, but map to same general message on UI as webauthn. I can bring this up to the team tomorrow